### PR TITLE
Keras cropping fixes

### PR DESCRIPTION
**Pull request by agibsonccc**
## What changes were proposed in this pull request?
Adds keras 2d cropping fixes.
(Please fill in changes proposed in this fix)

## How was this patch tested?
Manually tested with user model
(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ X] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [X ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [X ] Relevant tests for your changes are passing.

### CODE REVIEW

**Review by saudet — COMMENTED** [review]
I'm only seeing formatting changes here...

**Comment by agibsonccc** [comment]
@saudet  https://github.com/eclipse/deeplearning4j/pull/9204/commits/87cf665e22e2bb1296999ddd38ab5f84f5f7330a#diff-16cfefafb88a93df1f6b8b74b5c7731ecbb8ab3d56007499dda343cf9b2b0c41R373 Major change was an indexing issue with padding. It was missing getting the proper cropping number from keras. See:  getConvolutionModeFromConfig(Map<String, Object> l line 373

**Review by saudet — APPROVED** [review]
